runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -8406,7 +8408,23 @@ class INTERPRETER_CLASS
 
 		J9JNIMethodID *methodID = (J9JNIMethodID *)(UDATA)J9OBJECT_U64_LOAD(_currentThread, memberNameObject, _vm->vmindexOffset);
 		J9ROMMethod *romMethod = J9_ROM_METHOD_FROM_RAM_METHOD(methodID->method);
-		UDATA methodArgCount = romMethod->argCount;
+		UDATA methodArgCount = 0;
+
+		if (J9_ARE_ALL_BITS_SET(romMethod->modifiers, J9AccVarArgs)) {
+			j9object_t methodType = J9VMJAVALANGINVOKEMEMBERNAME_TYPE(_currentThread, memberNameObject);
+			/* +1 to account for the receiver */
+			methodArgCount = VM_VMHelpers::methodTypeParameterSlotCount(_currentThread, methodType) + 1;
+
+			if (J9_CLASS_FROM_METHOD(methodID->method) == J9VMJAVALANGINVOKEMETHODHANDLE(_currentThread->javaVM)) {
```

Comment:
you can use `_vm` here

Reply:
should use the `J9VMJAVALANGINVOKEMETHODHANDLE_OR_NULL` instead of `J9VMJAVALANGINVOKEMETHODHANDLE`. AFAIK at this point we should be guaranteed that MH is loaded